app.yml

```diff
@@ -0,0 +1,16 @@
+name: augur-node
```

Comment:
I'd encapsulate all this functionality in a script and put it in /scripts, see docket-entrypoint.sh.

Even better would be to have DO just run the docker image.